bubble.rb

```diff
@@ -0,0 +1,58 @@
+def bubble_sort(array)
+  i = 0
+  j = 0
+  not_sorted = true
+  while i < array.length - 1 && not_sorted
+    not_sorted = false
+    j = 0
+    while j < array.length - 1
+      if array[j].to_i > array[j + 1].to_i
+        temp = array[j]
+        array[j] = array[j + 1]
+        array[j + 1] = temp
+        not_sorted = true
+      end
+      j += 1
+    end
+    i += 1
+  end
+end
+
+def bubble_sort_by(array)
+  i = 0
```

Comment:
* [ ] bubble_sort_by method is implemented and working  ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_bubble_sort/sortBy.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-1))
currently for this 
```string_array = %w[hi hello hey another_string haloo testing]
p bubble_sort_by(string_array) { |string1, string2| string1.length <=> string2.length }example ```bubble_sort_by``` 
```
returns
``` 
["hi", "hey", "hello", "haloo", "testing", "another_string"]nil
``` 
look closely at the nil at the back of the sorted array. It ought not be there, kindly fix it.